### PR TITLE
6917 - Adjusted date and timepicker icons in datagrid filter

### LINKED ISSUE #6917: Datagrid: Datepicker is not shown on datagrid filter in Safari
**Describe the bug**
Datepicker icons are not shown in date column filters when in a Safari browser.

**To Reproduce**
1. Go to http://localhost:4000/components/datagrid/test-editable-datetime.html
2. See that the date column filters are missing datepicker icons

**Expected behavior**
It is expected that datepicker icons appear in the date column filters.

**Version**
- ids-enterprise: v4.68.0-dev

**Screenshots**
image

**Platform**
 - Infor Application/Team Name: lawson
 - OS Version: macOS Big Sur
 - Browser Name: safari
 - Browser Version: Version 14.1.2

**Additional context**
This broke in v4.56.0

### PR DESCRIPTION
**Pull request by yohannahbautista**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Adjusted position and sizes of date and timepicker for datagrid filter rows.

Note: Failed tests are due to icon changes, will update baseline images once changes are confirmed okay

**Related github/jira issue (required)**:

Closes #6917 

**Steps necessary to review your pull request (required)**:

- Pull this branch, build and run the app
- In Safari, go to: http://localhost:4000/components/datagrid/test-editable-datetime.html
- Datepicker and timepicker icons should appear in filter row

**Included in this Pull Request**:
- [ ] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
 - `[Datagrid]` Fixed a bug in datagrid where sorting is not rendering properly when there is a previously focused cell. ([#6851](https://github.com/infor-design/enterprise/issues/6851))
 - `[Datagrid]` Additional checks when updating cell so that numbers aren't converted twice. ([NG#1370](https://github.com/infor-design/enterprise-ng/issues/1370))
 - `[Datagrid]` Additional fixes on dirty indicator not updating on drag columns. ([#6867](https://github.com/infor-design/enterprise/issues/6867))
+- `[Datagrid]` Adjusted date and timepicker icons in datagrid filter. ([#6917](https://github.com/infor-design/enterprise/issues/6917))
```
Comment (ericangeles) [inline docs/CHANGELOG.md:31]: @yohannahbautista can you move this to 4.70?

**Comment by ericangeles** [comment]
Looks like you need to update the visual images https://github.com/infor-design/enterprise/actions/runs/3407461002/jobs/5667118442#step:8:2406

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Looks good for the most part @yohannahbautista but if you try 

- open safari
- go to http://localhost:4000/components/datagrid/test-editable-datetime.html?theme=new&mode=light&colors=0066D4
- change to extra small or small row height
- in those the icons still have the same issues but the other two row heights are fixed

**Review by tmcconechy — APPROVED** [review]

**Comment by tmcconechy** [comment]
Fix looks good now @yohannahbautista - but i think some images need updates...

**Review by ericangeles — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**